guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -34,6 +34,14 @@ Clients will remain supported.
 For more details, see the https://community.theforeman.org/t/drop-debian-11-ruby-2-7-and-nodejs-14-support-in-foreman-3-14/40503[removal RFC].
 endif::[]
 
+=== {awx} Parameter Change
+The `ansible_tower_fqdn` parameter has been removed and replaced with `ansible_tower_api_url`.
+This new parameter includes both the previous `ansible_tower_fqdn` value and the required API path for {awx}.
+
+By default, the API path is set to `/api/controller/v2`.
+If you are using an older {awx} version, update the API path manually to `/api/v2`.
```

Comment:
This might raise questions about which version you mean. I have no idea here either but I'm going to suggest something and hope you will correct me :) My point is to be more specific:

```suggestion
If you are using an {awx} version earlier than X.Y, update the API path manually to `/api/v2`.
```

Reply:
See previous discussion here https://github.com/theforeman/foreman-documentation/pull/3720#discussion_r2009961092

Reply:
I dont't quite understand this. Are you saying that it does not make any sense from a tech perspective to mention the version? (aka. declining the suggestion)? Please let me know; then I will merge as is.

Reply:
This is the Jira ticket if you need more context https://issues.redhat.com/browse/SAT-30761
The issue is that we have no mentioning of the AAP in our docs, only AAC (Ansible Automation Controller) for downstream, or AWX for upstream.
However, this change is required for AAP only. AWX remains without any changes.